app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -1,384 +1,357 @@
-
-        PrefUtils.increaseAppStartCount(this);
-
-        // Initialize Picasso
-        // When we clone mOkHttpClient, it will use all the same cache and everything.
-        // Only the interceptors will be different.
-        // We shouldn't have the below interceptor in other instances.
-        OkHttpClient.Builder picassoClient = mOkHttpClient.newBuilder();
-        picassoClient.addInterceptor(new PlusPersonDownloader(plusClient));
-
-        mPicasso = new Picasso.Builder(this)
-                .downloader(new OkHttp3Downloader(picassoClient.build()))
-                .memoryCache(new LruCache(this))
-                .build();
-
-        JodaTimeAndroid.init(this);
-
-        refWatcher = LeakCanary.install(this);
-
-        mOrganizerChecker = new OrganizerChecker(PrefUtils.prefs(this));
-
-        GoogleAnalytics.getInstance(this).setAppOptOut(PrefUtils.isAnalyticsEnabled(this));
-
-        // Init LastLocationFinder
-        mLocationFinder = new GingerbreadLastLocationFinder(this);
-        mLocationFinder.setChangedLocationListener(this);
-        updateLastLocation();
-
-        initTaggedEventSeries();
-    }
-
-
-    /**
-     * Init TaggedEventSeries.
-     */
-    private void initTaggedEventSeries() {
-
-        mTaggedEventSeriesList = new ArrayList<>();
-        //Add DevFest
-        addTaggedEventSeriesIfDateFits(new TaggedEventSeries(this,
-                R.style.Theme_GDG_Special_DevFest,
-                "devfest",
-                Const.DRAWER_DEVFEST,
-                Const.START_TIME_DEVFEST,
-                Const.END_TIME_DEVFEST));
-        //Add Women Techmakers
-        addTaggedEventSeriesIfDateFits(new TaggedEventSeries(this,
-                R.style.Theme_GDG_Special_Wtm,
-                "wtm",
-                Const.DRAWER_WTM,
-                Const.START_TIME_WTM,
-                Const.END_TIME_WTM));
-        //Add Android Fundamentals Study Jams
-        addTaggedEventSeriesIfDateFits(new TaggedEventSeries(this,
-                R.style.Theme_GDG_Special_StudyJams,
-                "studyjam",
-                Const.DRAWER_STUDY_JAM,
-                Const.START_TIME_STUDY_JAMS,
-                Const.END_TIME_STUDY_JAMS));
-        //Add IO Extended
-        addTaggedEventSeriesIfDateFits(new TaggedEventSeries(this,
-                R.style.Theme_GDG_Special_IOExtended,
-                "i-oextended",
-                Const.DRAWER_IO_EXTENDED,
-                Const.START_TIME_IOEXTENDED,
-                Const.END_TIME_IOEXTENDED));
-    }
-
-    private void addTaggedEventSeriesIfDateFits(@NonNull TaggedEventSeries taggedEventSeries) {
-        DateTime now = DateTime.now();
-        if (BuildConfig.DEBUG || (now.isAfter(taggedEventSeries.getStartDateInMillis())
-                && now.isBefore(taggedEventSeries.getEndDateInMillis()))) {
-            mTaggedEventSeriesList.add(taggedEventSeries);
-        }
-    }
-
-    private void migrate(int oldVersion, int newVersion) {

```

Comment:
And this method is removed. Alpha related code is moved to alpha buildType related source folder.